java/runtime/src/main/java/io/ray/runtime/runner/RunManager.java

```diff
@@ -30,13 +30,25 @@ public static void startRayHead(RayConfig rayConfig) {
     command.add("--redis-password");
     command.add(rayConfig.redisPassword);
     command.addAll(rayConfig.headArgs);
+    
+    String numCpus = System.getProperty("num-cpus");
```

Comment:
Can you also add the new args to ray/java/runtime/src/main/resources/ray.default.conf ?

Reply:
That sounds good. But I will add a new member variable in the RayConfig,  I want to submit a new PR specifically for refactoring this comment